### PR TITLE
saint-petersburg.ru

### PR DESCRIPTION
**Pull request by galeksandrp**

### CODE REVIEW

File: src/chrome/content/rules/saint-petersburg.ru.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<ruleset name="saint-petersburg.ru">
+	<target host="saint-petersburg.ru" />
```
Comment (J0WI) [inline src/chrome/content/rules/saint-petersburg.ru.xml]: m.saint-petersburg.ru

**Comment by J0WI** [comment]
There are some mixed content issues. Please check if they are breaking anything.
Please also document the issue on www.

**Comment by J0WI** [comment]
What about mixed content here?

**Comment by galeksandrp** [comment]
1 adv affected.

**Pull request merged**